webrtc/README.md

```diff
@@ -260,21 +262,50 @@ be the same. On mismatch the final Noise handshake MUST fail.
 
 ## Multiplexing
 
-After [Connection Security](#connection-security):
-
-1. On success of the authentication handshake _X_, the used datachannel is
-   closed and the plain WebRTC connection is used with its multiplexing
-   capabilities via datachannels.
-
-### Open Questions
-
-- Can we use WebRTC’s data channels in _Browser_ to multiplex a single
-  connection, or do we need to run an additional multiplexer (e.g. yamux) on top
-  of a WebRTC connection and WebRTC datachannel? In other words, does WebRTC
-  provide all functionality of a libp2p muxer like Yamux (e.g. flow control)?
-
-  Yes, with WebRTC's datachannels running on top of SCTP, there is no need for
-  additional multiplexing.
+Following [Connection Security](#connection-security).
+
+The WebRTC browser APIs do not support half-closing nor resets of streams.
```

Comment:
The above description is misleading. Thanks for the note. I pushed https://github.com/mxinden/specs/pull/1/commits/a60234cfb44fe060c8448d0de6e91958a57fedbc updating the section.

WebRTC's reset as well as the corresponding `RTCDataChannel.close()` reset both sending and receiving. We would like to support the former without the latter.

All that said, in the end I don't see this mechanism as particularly useful. I expect readers to read messages from the stream in order, and not do any buffering to the application above. Thus a receiver would receive the `RESET` after having passed all previous messages to the application already. Thus acting just as if it received a `FIN` flag, i.e. closing its read side, but not discarding any messages.

I don't have a strong opinion on adding `RESET`. I guess it doesn't hurt and might enable future optimizations.

The above is not to be generalized to the entire proposal. While I don't see much value in the `RESET` flag, there is value in being able to half-close a stream via the `FIN` and `STOP_SENDING` flag.

Does the above make sense @melekes? These are rather subtle semantics. Please comment in case my explanations and reasoning in the proposal don't make sense to you.